uiowa/uiowa
# Feature: SiteNow event templates uids_base

In this PR:
 
Resolves https://github.com/uiowa/uiowa/issues/1465. 

# How to test

`blt frontend`

- Add events paragraph block to a page in `uids_base`
- View list, grid, and masonry options and verify that they are working
- Add section background colors and verify that the events remain visible
- Test on mobile
